torch/csrc/jit/passes/quantization_patterns.h

```diff
@@ -219,6 +253,7 @@ graph(%a_quant, %b_scalar, %alpha):
        add_scalar_out,
        quantized_add_scalar_out,
        add_scalar_filter},
+      {"quantized::cat", cat, quantized_cat},
```

Comment:
is this change related to this PR?

Reply:
this is just moving around quantized cat